update_service/graph-data.adoc

```diff
@@ -0,0 +1,90 @@
+:context: update-service-graph-data
+[id="update-service-graph-data"]
+= Mirroring the {product-title} Update Service graph data
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+The {product-title} Update Service ingests data from two sources to construct its update recommendations:
+
+. A container image repository registry, from which it retrieves release images. To start using the {product-title} Update Service, you will need to ensure you have xref:../updating/updating-restricted-network-cluster.adoc[mirrored] at least two OpenShift update release payloads into container image registries reachable by the target clusters and the Update Service.
```

Comment:
`at least two OpenShift update release payloads into container image registries reachable by the target clusters` I do not think this correct. OSUS can start without any primary or secondary metadata and we have tested this in the past. Also having this type of prerequisite is not user friendly.

Reply:
I don't see a point in an Update Service that serves only empty graphs, and phrasing this as a prereq gives us space to talk about these data inputs.